lib/ansible/modules/cloud/vmware/vmware_guest_disk_facts.py

```diff
@@ -147,16 +148,64 @@ def gather_disk_facts(self, vm_obj):
                     summary=disk.deviceInfo.summary,
                     backing_filename=disk.backing.fileName,
                     backing_datastore=disk.backing.datastore.name,
-                    backing_disk_mode=disk.backing.diskMode,
-                    backing_writethrough=disk.backing.writeThrough,
-                    backing_thinprovisioned=disk.backing.thinProvisioned,
-                    backing_uuid=disk.backing.uuid,
-                    backing_eagerlyscrub=bool(disk.backing.eagerlyScrub),
                     controller_key=disk.controllerKey,
                     unit_number=disk.unitNumber,
                     capacity_in_kb=disk.capacityInKB,
                     capacity_in_bytes=disk.capacityInBytes,
                 )
+                if isinstance(disk.backing, vim.vm.device.VirtualDisk.FlatVer1BackingInfo):
+                    disks_facts[disk_index]['backing_type'] = 'FlatVer1'
+                    disks_facts[disk_index]['backing_writethrough'] = disk.backing.writeThrough
+
+                elif isinstance(disk.backing, vim.vm.device.VirtualDisk.FlatVer2BackingInfo):
```

Comment:
Please add example of these newly added keys in EXAMPLES section.

Reply:
I updated the existing (FlatVer2) and added "RawDiskMappingVer1"